pkg/tsdb/phlare/query_test.go

```diff
@@ -421,6 +463,20 @@ func Test_profileAsTree(t *testing.T) {
 	}
 }
 
+func Benchmark_profileAsTree(b *testing.B) {
+	wd, err := os.Getwd()
+	require.NoError(b, err)
+	profJson, err := os.ReadFile(filepath.Join(wd, "./testdata/profile_response.json"))
+	require.NoError(b, err)
+	var prof *googlev1.Profile
+	err = json.Unmarshal(profJson, &prof)
+	require.NoError(b, err)
+
```

Comment:
You probably want to `b.ResetTimer()` here.

Reply:
Good point does not seem to affect the benchmark much though.